Routers/userRouter.js

```diff
@@ -29,44 +31,51 @@ const router = express.Router();
           country: req.body.country,
           zip_code: req.body.zip_code,
           birthdate: req.body.birthdate,
-          date_registration: req.body.date_registration
+          date_registration: req.body.date_registration,
+          current_data_registration: req.body.current_data_registration
       }).save();
     
       res.send(user);
-      }
+    }
     catch (error) {
-      console.error(error);
+        console.error(error);
+        res.status(500).json({ error: "Internal Server Error" });
     }
   });
   
   /**Login**/
   router.post("/login", async (req, res) => {
     try {
-      const user = await User.findOne({ email: req.body.email });
-      console.log(req.body);
-      if (user) {
+        const user = await User.findOne({ email: req.body.email });
+        console.log(req.body);
```

Comment:
wszystkie console logi wyczysc szczegolnie tego co przychodzi nie jest to aktualnie potrzebne